app/test_main.py

```diff
@@ -1,3 +1,28 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_years,dog_years,result",
+    [
+        pytest.param(15, 15, [1, 1], id="exactly one year"),
+        pytest.param(24, 24, [2, 2], id="exactly two years"),
+        pytest.param(28, 29, [3, 3], id="exactly three years"),
+        pytest.param(32, 34, [4, 4], id="exactly four years"),
+        pytest.param(0, 0, [0, 0], id="zero years"),
+        pytest.param(7, 14, [0, 0], id="zero years - we discard the reminder"),
+        pytest.param(15, 20, [1, 1], id="1 year, we discard the reminder"),
+        pytest.param(27, 25, [2, 2], id="2 years, we discard the reminder"),
+        pytest.param(29, 33, [3, 3], id="3 years, we discard the reminder"),
+        pytest.param(33, 35, [4, 4], id="4 years, we discard the reminder"),
+        pytest.param(127, 100, [27, 17], id="big years value"),
+
+    ]
+)
+def test_convert_animal_years_to_human(cat_years: int,
+                                       dog_years: int,
+                                       result: list) -> None:
+    assert get_human_age(cat_years, dog_years) == result, (
+        (f"{cat_years} cat years and {dog_years} dog years "
+         f"are equal to {result} human years")
+    )
```

Comment:
Have a look to the requirements inside checklist:
1) you should have a test for the wrong type values like "str"
2) you should test negative values too
3) you should test work of function with laaaaaarge numbers))))
And check one more time that you have covered all edge cases